File: Example1/src/datastructure/Stack.java
package datastructure;

public class Stack {
    public Node head = null;

    public Stack() {
    }

    boolean isEmpty() {
        return this.head == null;
    }

    public void push(Node node) {
        if (this.isEmpty()) {
            this.head = node;
        } else {
            node.down = this.head;
            this.head = node;
        }

    }

    public Node pop() {
        if (this.isEmpty()) {
            System.out.println("Stack is empty");
            return null;
        } else {
            Node delete = this.head;
            this.head = this.head.down;
            return delete;
        }
    }
}

File: PushDownAutomataSample1/stack.py
class Stack():

    def __init__(self):
        self.arr = list()

    def isEmpty(self):
        if not self.arr:
            return True
        return False

    def addStack(self, value):
        self.arr.append(value)

    def delete(self):
        if self.isEmpty():
            print("Stack is empty")
        else:
            self.arr.pop()